src/generateResponseType.js

```diff
@@ -246,6 +254,7 @@ export const objectPropertiesToFlow = (
 
                 default:
                     config.errors.push(
+                        // eslint-disable-next-line flowtype-errors/uncovered
                         `Unsupported selection kind '${selection.kind}'`,
```

Comment:
Why did this become uncovered?

Reply:
Because I'm now handling all options! this default is now ~future-proofing